service/models.py

```diff
@@ -65,9 +65,8 @@ class Shopcart(db.Model):
     ##################################################
     id = db.Column(db.Integer, primary_key=True)
     user_id = db.Column(db.Integer)
-    create_time = db.Column(db.DateTime, nullable=False, server_default=str(datetime.utcnow()))
-    update_time = db.Column(db.DateTime, nullable=False, server_default=str(datetime.utcnow()),
-                            server_onupdate=str(datetime.utcnow()))
+    create_time = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    update_time = db.Column(db.DateTime, nullable=False, default=datetime.utcnow, onupdate=datetime.utcnow)
```

Comment:
This and line 187 are a bit too long so our pylint score dropped a little

Reply:
reformatted :)